src/lib/components/dashboard/Section.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+  import TokenStreams from 'radicle-design-system/icons/TokenStreams.svelte';
+
+  export let title: string;
+  export let earningPerDay: number | undefined = undefined;
+  export let count: number | undefined = undefined;
+</script>
+
+<div class="section">
+  <div class="headline">
+    <div class="title">
+      <h2>{title}</h2>
+      {#if count}
+        <div class="count"><h3>{count}</h3></div>
+      {/if}
+    </div>
+    {#if earningPerDay}
+      <div class="earning-per-day">
+        <TokenStreams />
+        <p>
+          You are earning <span class="typo-text-bold">{earningPerDay} DAI</span
+          > / day
+        </p>
+      </div>
+    {/if}
```

Comment:
Shouldn't this just be a `<Slot />` Seems more flexible/general purpose to me.

Reply:
👍👍 a named slot would make sense here. Will adjust that

Reply:
Nice, also is there a way that prettier converts all `px` to `rem` on save or something?

Reply:
That'd be great, but couldn't find any with a short Google :D I suppose I just need to get used to that, just keep pointing it out ;)